ospd_openvas/nvticache.py

```diff
@@ -143,7 +143,8 @@ def get_nvt_refs(oid):
 
 def get_nvt_prefs(ctx, oid):
     """ Get NVT preferences. """
-    prefs = ctx.smembers('oid:%s:prefs' % oid)
+    key = ('oid:%s:prefs' % oid)
```

Comment:
I suppose this isn't a python tuple so please remove the braces. If it should be a tuple please a a colon at the end.